src/frontend/js/components/LtiConsumer/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect } from 'react';
+import { iframeResizer } from 'iframe-resizer';
+import { LtiConsumer as LtiConsumerProps } from 'types/LtiConsumer';
+
+export const LtiConsumer = ({ url, contentParameters }: LtiConsumerProps) => {
+  const formRef = React.useRef<HTMLFormElement>(null);
+  const iframeRef = React.useRef<HTMLIFrameElement>(null);
+
+  useEffect(() => {
+    formRef.current?.submit();
+    iframeResizer({ sizeWidth: true }, iframeRef.current!);
+  }, []);
+
+  return (
+    <div className="lti-consumer">
+      <form
+        id="lti_form"
+        ref={formRef}
+        action={url}
+        method="POST"
+        encType="application/x-www-form-urlencoded"
+        target="lti_iframe"
+        style={{ display: 'none' }}
+      >
+        {Object.entries(contentParameters).map(([name, value]) => {
+          return <input key={name} type="hidden" name={name} value={value} />;
+        })}
```

Comment:
Just to show a little shortcut.
```suggestion
        {Object.entries(contentParameters).map(([name, value]) => (
          <input key={name} type="hidden" name={name} value={value} />
        ))}
```